docs/README.rst

```diff
@@ -4,6 +4,8 @@
 .. image:: https://readthedocs.org/projects/adaptdl/badge/?version=latest
   :target: https://adaptdl.readthedocs.io/en/latest/?badge=latest
   :alt: Documentation Status
+.. image:: https://codecov.io/gh/petuum/adaptdl/branch/master/graph/badge.svg
+  :target: https://codecov.io/gh/petuum/adaptdl
```

Comment:
Put before the docs badge, ordering can be build, coverage, docs.

Reply:
Fixed, see https://github.com/petuum/adaptdl/blob/d4fc685e451c25f4735016cf0e043da9359eef22/docs/README.rst

Also added build and pypi status.